### PR TITLE
chore: add playmode tests

### PR DESCRIPTION
**Pull request by kuruk-mm**
## What does this PR change?

Resolve decentraland/unity-renderer#1173

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

**Comment by kuruk-mm** [comment]
Even the tests are failing, the task is to enable the playmode-tests, not to fix those.

So this is ready for review.

File: unity-renderer-desktop/Assets/Scripts/Editor/UnitTestCommand.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+using System;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEditor.Build.Reporting;
+
+static class UnitTestCommand
+{
+    static void ExecuteTests()
+    {
+        Console.WriteLine(":: Executing Tests");
```
Comment (kuruk-mm) [inline unity-renderer-desktop/Assets/Scripts/Editor/UnitTestCommand.cs:10]: This is wanted? Maybe we can delete this class

File: unity-renderer-desktop/Packages/packages-lock.json [hunk 2]
```diff
@@ -14,7 +14,7 @@
       "dependencies": {}
     },
     "com.decentraland.unity-renderer": {
-      "version": "git+https://github.com/decentraland/unity-renderer.git?path=unity-renderer/Assets#master",
+      "version": "git+https://github.com/decentraland/unity-renderer.git?path=unity-renderer/Assets#fix/playmode-tests-desktop",
```
Comment (kuruk-mm) [inline unity-renderer-desktop/Packages/packages-lock.json:17]: Reminder: Merge unity-renderer PR first, update to master and then merge this PR

**Review by Kinerius — APPROVED** [review]

File: unity-renderer-desktop/Packages/packages-lock.json (hunk 2 above)
Comment (Kinerius) [reply unity-renderer-desktop/Packages/packages-lock.json:17]: We have to merge [this](https://github.com/decentraland/unity-renderer/pull/1505) first

File: unity-renderer-desktop/Assets/Scripts/Editor/UnitTestCommand.cs (hunk 1 above)
Comment (Kinerius) [reply unity-renderer-desktop/Assets/Scripts/Editor/UnitTestCommand.cs]: Done

**Pull request merged**